File: unix/boot/spp/rpp/rpprat/entdef.r
#-h-  entdef			  387  local   12/01/80  15:53:51
# entdef - enter a new symbol definition, discarding any old one
   include  defs

   subroutine entdef (name, defn, table)
   character name (MAXTOK), defn (ARB)
   pointer table

   integer lookup

   pointer text
   pointer sdupl

   if (lookup (name, text, table) == YES)
      call dsfree (text)   # this is how to do UNDEFINE, by the way
   call enter (name, sdupl (defn), table)

   return
   end

File: unix/boot/spp/rpp/rppfor/entdef.f
      subroutine entdef (name, defn, table)
      integer name (100), defn (100)
      integer table
      integer lookup
      integer text
      integer sdupl
      if (.not.(lookup (name, text, table) .eq. 1))goto 23000
      call dsfree (text)
23000 continue
      call enter (name, sdupl (defn), table)
      return
      end
